cpp/src/barretenberg/stdlib/primitives/safe_uint/safe_uint.hpp

```diff
@@ -135,7 +142,9 @@ template <typename ComposerContext> class safe_uint_t {
                 return std::make_pair((uint256_t)(val / (uint256_t)divisor), (uint256_t)(val % (uint256_t)divisor));
             })
     {
```

Comment:
I wasn't super careful thinking through the logic of safe_uint here, I was just trying to make tests pass.